docs/manual/java/guide/build/EclipseRunConfig.md

```diff
@@ -0,0 +1,21 @@
+# Creating an Eclipse Run Configuration
```

Comment:
👍 

Also, in Eclipse it's very natural to right click an item in the package explorer and run directly from there. In our case it'd be:

 * locate your project parent folder in your `Package Explorer`
 * select the option `Run as...` and then choose `Maven build...`
 * on the dialog you should specify the goal `lagom:runAll` 

that has the advantage of removing the most error-prone step of manually selecting the base directory.